src/main/cli/commandLine.ts

```diff
@@ -0,0 +1,120 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { readerActions } from "readium-desktop/common/redux/actions";
+import { container } from "readium-desktop/main/di";
+import { extractHostname } from "readium-desktop/utils/hostname";
+import { Store } from "redux";
+import * as yargs from "yargs";
+
+import { OpdsFeedRepository } from "readium-desktop/main/db/repository/opds";
+import { PublicationRepository } from "readium-desktop/main/db/repository/publication";
+import { RootState } from "readium-desktop/main/redux/states";
+import { CatalogService } from "readium-desktop/main/services/catalog";
+
+export interface ICliParam {
+    readonly argv: yargs.Arguments;
+    quit: boolean;
+}
+
+export interface ICli {
+    name: string;
+    fct: (o: ICliParam) => Promise<any>;
+    help: string[];
+}
+
+export const cli: ICli[] = [
+    {
+        name: "_",
+        fct: async ({ argv }) => {
+            let notRead = true;
+
+            argv._.map(async (path) => {
+                // import and read publication
+                const catalogService = container.get("catalog-service") as CatalogService;
+                const publication = await catalogService.importFile(path as string);
+                const store = container.get("store") as Store<RootState>;
+                if (publication && notRead) {
+                    store.dispatch({
+                        type: readerActions.ActionType.OpenRequest,
+                        payload: {
+                            publication: {
+                                identifier: publication.identifier,
+                            },
+                        },
+                    });
+                    notRead = false;
+                }
+                if (path !== "." && !publication && notRead) {
+                    console.error("Publication error, path:", path);
+                }
+            });
+        },
+        help: [],
+    },
+    {
+        name: "import",
+        fct: async ({ argv }) => {
+            const catalogService = container.get("catalog-service") as CatalogService;
+            return await catalogService.importFile(argv.import as string);
+        },
+        help: [
+            "--import PATH",
+            "import epub or lpcl file",
```

Comment:
Must this be an **absolute** path on the filesystem? If not, is the **relative** path based on `process.cwd()`? (i.e. the command line `cwd` from where the app executable is invoked)

Reply:
@panaC said:
> The path must be absolute
https://github.com/readium/readium-desktop/pull/403#issuecomment-512746474

Reply:
Okay, so can we please have this information in the help text, for example "import epub or lpcl file (must be absolute path)"